lib/workers/repository/process/lookup/index.ts

```diff
@@ -495,3 +472,85 @@ export async function lookupUpdates(
   }
   return res;
 }
+
+function addReplacementUpdateIfValid(
```

Comment:
Move this new functions in a utility or replacement file.

Reply:
Good suggestion, update pushed